### ISSUE TITLE
Improve error handling when starting log streaming

### ISSUE THREAD

**Issue by EricJizbaMSFT** [Microsoft/vscode-azurefunctions#266]
Usually log streaming works for me - but one time it didn't start and I didn't see any error message. So first of all - we need to make sure error message are displayed.

Second, we need to let the user 'restart' log streaming. Once I was in the error state - if I tried to start log streaming again I got this message:
![screen shot 2018-03-08 at 11 55 46 am](https://user-images.githubusercontent.com/11282622/37173261-ec739f86-22c7-11e8-86b5-17d5bd59666b.png)

And if I tried to stop log streaming I got this message:
![screen shot 2018-03-08 at 11 56 01 am](https://user-images.githubusercontent.com/11282622/37173274-f49681e2-22c7-11e8-8c6c-b8d3099fd404.png)

In other words - I was stuck and had to restart VS Code to make it work
